### DISCUSSION TITLE
Scrum Prep Milestone 1 -Week 3

### DISCUSSION THREAD

**Discussion by AngelUPR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#59] [Show and tell]
In one or two sentences,describe what you have done this week

**Comment by estefania-2817** [comment]
This week I learned how to use the Svelte framework and worked on coding the user interface for the sign up feature. Where I created the fields of studentID, email and password for student accounts and organization name, email and password for organization accounts.

**Comment by SammyAlva04** [comment]
This week I worked entirely on learning Svelte in order to use in for the Calendar feature.

**Comment by GabyMarr** [comment]
This week I worked on learning Svelte through [Svelte Interactive Tutorial](https://learn.svelte.dev/tutorial/welcome-to-svelte)

**Comment by KennethSepu1** [comment]
This week I was practicing my skills using HTML, CSS and JavaScript. I also learned how to use Svelte

**Comment by MiguelGibo** [comment]
Was assigned task by team leader and started learning Svelte.

**Comment by AlejandroClaudio** [comment]
Finished the Svelte tutorial and discussed tasks for sign up page.

**Comment by Ingrid1089** [comment]
This week I worked on learning Svelte.

**Comment by kevin-d-lopez** [comment]
Worked on svelte using the interactive tutorial and finished helping on the initial tag doc with all the tags planed to be used on the feature.

**Comment by EdzelOrtiz** [comment]
This week I was working on research for the tags organization by Faculty, Major, and finding all the official University associations.

**Comment by cseba11** [comment]
This week, I completed the Svelte tutorial and created the CalendarPage.svelte component. After that, I developed the Ribbon component to handle the Days, Weeks, and Months display on the Calendar page.
